src/team_lib/hier/xccl_hier_sbgp.c

```diff
@@ -320,9 +323,7 @@ xccl_status_t sbgp_create(xccl_hier_team_t *team, sbgp_type_t type)
         break;
     case SBGP_NODE_LEADERS:
         assert(SBGP_DISABLED != team->sbgps[SBGP_NODE].status);
-        if (team->sbgps[SBGP_NODE].status == SBGP_ENABLED) {
```

Comment:
oh, crap. plz add this conditional (team->sbgps[SBGP_NODE].status == SBGP_ENABLED) to the next "case" - SOCKET_LEADERS (not node leaders).